### ISSUE TITLE
Failed to update auto layout status: The agent crashed

### ISSUE THREAD

**Issue by emptyway** [LeonardoCardoso/DownloadButton#1]
After I assigned NFDownloadButton to a UIView, I get this error:

```
IB Designables: Failed to update auto layout status: The agent crashed
```

And If i want to create it programmatically, get this error: 

```
'NFDownloadButton' initializer is inaccessible due to 'internal' protection level
```

**Comment by leonardocardoso** [comment]
This is a known issue. It's set as fixed in the next version. Thanks.

**Comment by leonardocardoso** [comment]
Hey, the initializer thing is fixed in the version [0.0.2](https://github.com/LeonardoCardoso/NFDownloadButton/releases/tag/0.0.2). Regarding the InterfaceBuilder issue, there are some threads about it, have a look:

https://stackoverflow.com/questions/26799054/failed-to-render-instance-of-ib-designables#answer-26812639
https://stackoverflow.com/questions/31265906/ibdesignable-crashing-agent

I'll spend more time on it properly when I have time.

**Comment by emptyway** [comment]
@LeonardoCardoso  thanks for the prompt reply and for this awesome pod! 

I end up using the storyboard way, it show the error message but still compile and seems to work as expected.

**Comment by leonardocardoso** [comment]
@simonlinj I'm not sure but it looks like a Xcode bug, because I'm using GraphicsContext and Xcode can't render it properly on Storyboard. I may be wrong though.

**Comment by ghost** [comment]
Having the same issue, it isn't rendered in storyboard and throws these errors - but runs fine.

**Comment by Z--EN** [comment]
The same, XCode 9.2

**Comment by jr00n** [comment]
Any updates? Still an issue here in XCode 10.1

**Comment by leonardocardoso** [comment]
This seems to be a Xcode problem.

**Comment by realityworks** [comment]
Yes, has been around since Xcode 8. It's truly disappointing the engineers at Apple still have yet to fix this issue on complex StoryBoard/View layouts. Poor form Apple!
